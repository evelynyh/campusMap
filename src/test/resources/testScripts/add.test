CreateGraph graph1
AddNode graph1 n1
AddNode graph1 n2

CreateGraph graph2
ListNodes graph2

AddNode graph1 n3
AddEdge graph1 n3 n1 e1
AddEdge graph1 n2 n1 e2

AddNode graph2 n1
AddNode graph2 n2
AddEdge graph2 n2 n1 e1
AddNode graph2 n2

ListNodes graph1
ListChildren graph1 n3
ListChildren graph1 n3