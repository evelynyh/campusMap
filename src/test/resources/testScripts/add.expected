created graph graph1
added node n1 to graph1
added node n2 to graph1

created graph graph2
graph2 contains:

added node n3 to graph1
added edge e1 from n3 to n1 in graph1
added edge e2 from n2 to n1 in graph1

added node n1 to graph2
added node n2 to graph2
added edge e1 from n2 to n1 in graph2
added node n2 to graph2

graph1 contains: n1 n2 n3
the children of n3 in graph1 are: n1(e1)
the children of n3 in graph1 are: n1(e1)
